### PR TITLE
executor/inspect: Add a `status_address` of `inspection_result` table to indicate the `status address`.

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

Before this PR, the `instance` value of `inspection_result` is wrong. The value in some row is `grpc service address`, such as `127.0.0.1:4000`, the value in some row is `status address`, such as `127.0.0.1:10080`. 

* Fix `instance` value.
* Add a `status_address` of `inspection_result` table to indicate the `status address`.

#### Why add a `status_address`?

Because some diagnose rule such as `server down`,  we can only get the `status address` from Prometheus. 

### Check List 

Tests 

- Unit test

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM

### Release note 

- Add a `status_address` of `inspection_result` table to indicate the `status address`.

### CODE REVIEW

File: executor/inspection_result.go [hunk 1]
```diff
@@ -482,13 +510,13 @@ func (criticalErrorInspection) inspectError(ctx context.Context, sctx sessionctx
 				result := inspectionResult{
 					tp: rule.tp,
 					// NOTE: all tables which can be inspected here whose first label must be `instance`
```
Comment (reafans) [inline executor/inspection_result.go:512]:
```suggestion
					// NOTE: all tables which can be inspected here whose first label must be `statusAddress`
```

Comment (crazycs520) [reply executor/inspection_result.go:512]: The label name of metric is `instance`, but the value is status address.

**Pull request merged**